website/docs/docs/cloud/manage-access/auth0-migration.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Migrating to Auth0 for SSO"
+id: "auth0-migration"
+sidebar: "SSO Auth0 Migration"
+description: "Required actions for migrating to Auth0 for SSO services on dbt Cloud"
+---
+
+dbt Labs is partnering with Auth0 to bring enhanced features to dbt Cloud's single sign-on (SSO) capabilities. Auth0 is an identity and access management (IAM) platform with advanced security features, and it will be the dbt Cloud SSO identity provider (IDP) moving forward. These changes will require some action from customers with SSO configured in dbt Cloud today, and this guide will outline the necessary changes for each environment. 
+
+If you have not yet configured SSO in dbt Cloud, refer instead to our setup guides for [SAML](docs/cloud/manage-access/set-up-sso-saml-2.0), [Okta](docs/cloud/manage-access/set-up-sso-okta), [Google Workspace](docs/cloud/manage-access/set-up-sso-google-workspace), or [Azure Active Directory](docs/cloud/manage-access/set-up-sso-azure-active-directory) single sign-on services.
+
+## SAML 2.0 and Okta
+
+SAML 2.0 users must update a few fields in the SSO app configuration to match the new Auth0 URL and URI.  You can approach this by editing the existing SSO app settings or creating a new one to accommodate the Auth0 settings. One approach isn't inherently better, so choose whichever works best for your organization.
+
+The fields that will be updated are:
+- Single sign-on URL &mdash; `https://us-devspace-cluster.us.auth0.com/login/callback?connection={slug}`
+- Audience URI (SP Entity ID) &mdash; `urn:auth0:us-devspace-cluster:{slug}`
+
+Replace {slug} with your organization’s login slug. It must be unique across all dbt Cloud instances and is usually something like your company name separated by dashes (for example, `dbt-labs`).
+
+Here is an example of an updated SAML 2.0 setup in Okta.
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/new-okta-config.png" title="Okta configuration with new URL"/>
+
+After the configuration is saved, your SAML settings should look something like this:
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/new-okta-completed.png" title="New Okta configuration completed"/>
+
+Once you have saved this information in the SSO environment, you must edit some information in the dbt Cloud settings. Navigate to the **Account Settings**, update the single sign-on URL fields, and provide the updated x.509 certificate.
+
+Make sure to enable the "Use Auth0" option to ensure that the traffic is routed correctly. _The Auth0 migration action is final and cannot be undone_
```

Comment:
Just want to check/confirm what the plan is if a customer accidentally toggle this. Both for communication (i.e they need to reach out support), and for the playbook on our end. Hopefully this doesn't happen, but odds are high enough it may occur once or twice.